.github/workflows/ci.yml

```diff
@@ -0,0 +1,25 @@
+name: ci
+
+on:
+  push:
+    branches: [ master ]
+    tags:
+      - 'v*'
+
+jobs:
+
+  build:
+    if: startsWith(github.ref, 'refs/tags/v')
```

Comment:
do we need to have this check together with `tags: - 'v*'` ?

Reply:
fixed in [6b72385](https://github.com/Jimdo/yaml-vault/pull/3/commits/6b723853f7c81121db551305d600c7db544d5c88)